.github/workflows/win.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         run: |
           git config --global core.autocrlf false
       - name: Enable Developer Command Prompt
-        uses: ilammy/msvc-dev-cmd@ed94116c4d30d2091601b81f339a2eaa1c2ba0a6
+        uses: ilammy/msvc-dev-cmd@674ff850cbd739c402260838fa45b7114f750570
```

Comment:
I think it's updating to latest master, not to whatever the latest release is. It might require a comment like the other action below. We should do that first.

Reply:
Let's try with #12746

Reply:
Yeah, that worked as expected.